cross_contract/contract_b/Makefile

```
@@ -0,0 +1,24 @@
+CUR_DIR = $(shell pwd)
+default: build
+
+all: build test
+
+test: build
+	cargo test
+	cargo test --features testutils
+
+build:
+	@cd ../contract_a && make build || break; 
+
+	@cd $(CUR_DIR);
```

I think you can run make in an alternative directory. I think it's common practice to use `$(MAKE)` instead of `make` too, although I'm fuzzy on the why.
```suggestion
	$(MAKE) -C ../contract_a build || break; 
```